tap_partnerize/client.py

```diff
@@ -134,18 +183,36 @@ def parse_response(self, response: requests.Response) -> Iterable[dict]:
         Yields:
             Each record from the source.
         """
-        # TODO: Parse response body and return a set of records.
-        yield from extract_jsonpath(self.records_jsonpath, input=response.json())
 
-    def post_process(self, row: dict, context: dict | None = None) -> dict | None:
-        """As needed, append or transform raw data to match expected structure.
-
-        Args:
-            row: An individual record from the stream.
-            context: The stream context.
-
-        Returns:
-            The updated record dictionary, or ``None`` to skip the record.
-        """
-        # TODO: Delete this method if not needed.
-        return row
+        count = 0
+        f = io.StringIO(response.text)
+        for row in csv.DictReader(f):
```

Comment:
```suggestion
        for count, row in enumerate(csv.DictReader(f)):
```
i think this would give you the incrementing `count` variable